command.go

```diff
@@ -328,12 +328,17 @@ func stripFlags(args []string, c *Command) []string {
 	return commands
 }
 
-func argsMinusX(args []string, x string) []string {
+// argsMinusFirstX removes only the first x from args.  Otherwise, commands that look like
+// openshift admin policy add-role-to-user admin my-user, lose the admin argument (arg[4]).
+func argsMinusFirstX(args []string, x string) []string {
+	removedFirstX := false
 	newargs := []string{}
 
 	for _, y := range args {
-		if x != y {
+		if removedFirstX || (x != y) {
 			newargs = append(newargs, y)
+		} else {
+			removedFirstX = true
```

Comment:
> What do you think?

I'm not against it, except for it not working: 
```
[deads@deads-dev-01 cobra]$ go test
Error: unknown command "one"
Run 'print help' for usage.
--- FAIL: TestChildSameName (0.00 seconds)
	cobra_test.go:345: Wrong command called
	cobra_test.go:348: Command didn't parse correctly
```

I checked in a similar version that passes.